src/dispatchers/mouseDispatcher.ts

```diff
@@ -191,11 +188,11 @@ export class Mouse extends Dispatcher {
       throw new Error("Invalid scope '" + scope + "', must be 'element' or 'page'");
     }
     if (scope === "page" || this.eventInside(component, event)) {
-      const newMousePosition = this._translator.computePosition(event.clientX, event.clientY);
-      if (newMousePosition != null) {
-        this._lastMousePosition = newMousePosition;
-        this._callCallbacksForEvent(eventName, this.lastMousePosition(), event);
-      }
+      const origin = component.originToRoot();
```

Comment:
can't delete because it's used for touch events. see follow up PR for perf update there.